include/llvm/Support/WinTypes.h

```diff
@@ -1,6 +1,20 @@
 #ifndef LLVM_SUPPORT_WINDTYPES_H
 #define LLVM_SUPPORT_WINDTYPES_H
 
+// The following macros are defined to facilitate the lack of 'uuid' on Linux.
+#ifdef _WIN32
```

Comment:
The `_WIN32` part should be moved into `dxcapi.h` itself. So for Windows, developers just need to pull in `dxcapi.h` itself. Putting here will make the `dxcapi.h` depend on `WinType` always. I think it will break for Windows since this header is only included on non-Windows platforms.